modules/nw-rfhe-installing-operator-cli.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * monitoring/using-rfhe.adoc
+
+:_content-type: PROCEDURE
+[id="nw-rfhe-installing-operator-cli_{context}"]
+= Installing the Baremetal Hardware Event Proxy Operator using the CLI
+
+As a cluster administrator, you can install the Operator by using the CLI.
+
+.Prerequisites
+
+* A cluster installed on bare-metal hardware with nodes that have Redfish hardware.
```

Comment:
Flagging this sentence for inconsistent use of "bare-metal". In other parts of Redfish doc you have used "baremetal".